libs/pageserver_api/src/models.rs

```diff
@@ -115,7 +115,9 @@ impl TenantState {
             // It will run a Postgres-level health check as soon as it observes Attached.
             // That will fail on Broken tenants.
             // Console can then rollback the attach, or, wait for operator to fix the Broken tenant.
-            Self::Broken { .. } => Attached,
+            Self::Broken { reason, .. } => Failed {
+                reason: reason.to_owned(),
+            },
```

Comment:
The commet above this arm is now out of date with:

```
We would need a separate TenantState variant to distinguish these cases.
However, there's no practical difference from Console's perspective.
```

I am still unsure what is the practical difference?

Wasn't there something on the thread about this as well:

```
It will run a Postgres-level health check as soon as it observes Attached.
That will fail on Broken tenants.
```

Reply:
Ok found the practical difference from the openapi.yaml after reading it a bit more.

Ok, this change now starts to make sense, I think we want to cleanup this comment regardless.

Reply:
Updated the comment, please take a look, resolve if its ok

Reply:
I think we can just get rid of the last line comment
```
            // If needed BrokenFrom can be used akin to ActivatingFrom above.
```

Reply:
Done